plugin/adsk/scripts/AEmayaUsdProxyShapeBaseTemplate.mel

```diff
@@ -40,15 +40,39 @@ global proc AEmayaUsdProxyShapePayloadsNew(string $loadPayloads)
     AEmayaUsdProxyShapePayloadsReplace($loadPayloads);
 }
 
+global proc AEMayaUsdProxyShapeStageLoadAll(string $fullNodeName)
+{
+    mayaUsdStageLoadAll $fullNodeName;
+
+    // Maya 2022 and above will automatically refresh the viewport, but
+    // for earlier versions of Maya we need to do it manually.
+    int $mjv = `about -majorVersion`;
+    if ($mjv < 2022) {
+        refresh;
+    }
```

Comment:
Special case for Maya 2020 (and earlier) since they don't have UFE v2 which contains the subtree invalidate notif that we use to refresh Maya viewport.